neo/Consensus/ConsensusContext.cs

```diff
@@ -143,7 +143,7 @@ public uint GetPrimaryIndex(byte viewNumber)
 
         public bool Load()
         {
-            byte[] data = store.Get(CN_Context, new byte[0]);
+            byte[] data = store.Get(ConsensusStateKey);
```

Comment:
Is not possible to move this to Store and remove Get, Put and PutSync, something like... ?

```csharp
public abstract DataCache<byte[], byte[]> GetRandomMemory();
```

And use the snapshot, instead of this methods?

Reply:
We need `PutSync()`.